docker/Readme.md

```diff
@@ -0,0 +1,39 @@
+# Description:
+
+Files in this folder is used to make docker image of intel/kinetic:moveit_handson base environment.
+
+Dockerfile -- the base file which will be automatically loaded when run docker build command
+
+setup_docker_display.sh -- set the container to display a pop-up
+
+# Build command:
+First, clone the repo to your local system.
+```
+sudo docker build -t intel/kinetic:moveit_handson .
+```
+If proxies are needed:
+```
+sudo docker build -t intel/kinetic:moveit_handson --build-arg http_proxy=http://proxy.my.com:### --build-arg https_proxy=http://proxy.my.com:### .
+```
+# Run docker image
+After the project runs, there will be a pop-up window, you need to set the operating environment first.
+```
+./setup_docker_display.sh
+```
+Run this docker images
+```
+
+sudo docker run -t -i --rm -v $XSOCK:$XSOCK:rw -v $XAUTH:$XAUTH:rw -e XAUTHORITY=${XAUTH} -e DISPLAY --name moveit_handson intel/kinetic:moveit_handson bash
+or

```

Comment:
\# or

Reply:
I think we should add some explanation about why there is two options.

Reply:
Not sure if docker will search the image automatically from the disk or we have to specify a path to the image?

Reply:
The second command I wrote was used for the new terminal after running the install script. Of course this is not required, I can delete it.
I can package the image and use the Dockerfile for easy sharing and updates.